Composer/packages/adaptive-form/src/components/ExpressionSwitchWindow.tsx

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core';
+import { FontIcon } from 'office-ui-fabric-react/lib/Icon';
+import React from 'react';
+import formatMessage from 'format-message';
+
+const styles = {
+  fieldTypeText: css`
+    height: 32px;
+    font-style: italic;
+    color: #8a8886;
+    padding: 0 4px;
+    width: 100%;
+    display: flex;
```

Comment:
Why do you need flex here?

Reply:
@GeoffCoxMSFT, in order to address this:
![image](https://user-images.githubusercontent.com/66701106/96322735-2185dd00-0fcf-11eb-902a-77382bc0ecf0.png)
I could wrap the two divs inside another div but this would add unnecessary complexity to the dom

Reply:
display: flex affects the items inside the element, not the element itself - it will be the equivalent of block.  If you put flex on a div that contains string content, it doesn't going to do anything.